Doc/library/decimal.rst

```diff
@@ -1570,7 +1570,7 @@ condition.
    Altered an exponent to fit representation constraints.
 
    Typically, clamping occurs when an exponent falls outside the context's
-   :attr:`Emin` and :attr:`Emax` limits.  If possible, the exponent is reduced to
+   :attr:`!Emin` and :attr:`!Emax` limits.  If possible, the exponent is reduced to
```

Comment:
```suggestion
   :attr:`~Context.Emin` and :attr:`~Context.Emax` limits.  If possible, the exponent is reduced to
```

As mentioned, these are valid attributes that should be documented at some point.